app/components/LegoReactions/index.tsx

```diff
@@ -58,32 +63,51 @@ const LegoReactions = (props: Props) => {
     });
   }
 
+  let usersByReaction = {};
+
+  if (parentEntity.reactionsGrouped && parentEntity.reactions) {
+    for (let groupedReaction of parentEntity.reactionsGrouped) {
+      for (let reaction of parentEntity.reactions) {
+        if (reaction.emoji === groupedReaction.emoji) {
```

Comment:
This check is done for every combination of reactions and grouped reactions, even though emojis from reactions could be directly added to the usersByReaction without needing to check against groupedReaction.